tools/report.sh

```diff
@@ -301,11 +301,11 @@ done
 echo "events"
 EVENTS=$($KUBE_CLIENT get event -n "$NAMESPACE" --ignore-not-found) && readonly EVENTS
 if [[ -n $EVENTS ]]; then
-  mkdir -p "$OUT_DIR"/reports/events
-  echo "$EVENTS" > "$OUT_DIR"/reports/events/events.txt
+  mkdir -p "$OUT_DIR"/reports/logs
+  echo "$EVENTS" > "$OUT_DIR"/reports/logs/events.txt
```

Comment:
Events are not logs. Maybe a separate directoy makes sense?

Reply:
Well, they have a timestamp, a log level and a message so I thought it could be a good place for it. Additionally, you may want to have a look at them when checking logs. That said, if you prefer to have them in a dedicated dir, then I'm fine.

Reply:
I would agree with Jakub, let's keep events in a dedicated folder.

Reply:
ok